llabake/Hello-Books
# chore(api documentation): update API documentation

#### What does this PR do?
This PR documents the API for the application

#### Description of Task to be completed?
  *  update Readme file with front-end test script
   * Document all API endpoint
#### How should this be manually tested?
* clone the repo
* cd into the project directory
* run `npm install` to install all the dependency
* run `npm run start-dev` to start the back-end server
* navigate to `localhost:5000/api-docs/`
* the API documentation should be there
#### What are the relevant pivotal tracker stories?

[finishes #158409247]

#### Any background context you want to add?
N/A

#### Screenshots

N/A
